checker/src/main/java/org/checkerframework/checker/nullness/KeyForSubchecker.java

```diff
@@ -9,4 +9,11 @@
  * @checker_framework.manual #map-key-checker Map Key Checker
  * @checker_framework.manual #nullness-checker Nullness Checker
  */
-public class KeyForSubchecker extends BaseTypeChecker {}
+public class KeyForSubchecker extends BaseTypeChecker {
+    {
+        // While strictly required for soundness, this leads to too many false positives.  Printing
+        // a key or putting it in a map erases all knowledge of what maps it was a key for.
+        // TODO: Revisit when side effect annotations are more precise.
+        // this.sideEffectsUnrefineAliases = true;
```

Comment:
This needs to move, too.

Reply:
Thanks for catching that.  I found one other reference that needed to be changed, too.